### PR TITLE
Reduce churn of organization when no-op entry processors are executed.

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/organizations/HazelcastOrganizationService.kt [hunk 1]
```diff
@@ -523,34 +539,37 @@ class HazelcastOrganizationService(
...
+            val modified = it.members.remove(principal)
+            Result(modified, modified)
+        })
     }
 
     fun updateRoleGrant(organizationId: UUID, roleId: UUID, grant: Grant) {
         organizations.executeOnKey(organizationId, OrganizationEntryProcessor {
             it.grants.getOrPut(roleId) { mutableMapOf() }[grant.grantType] = grant
-            null
+            Result(null) //TODO: Being lazy not implementing diff as this should rarely be called.
```
Comment (solomontang) [inline src/main/kotlin/com/openlattice/organizations/HazelcastOrganizationService.kt:551]: Looks like the build is failing without handling this case tho

```OrganizationEntryProcessorStreamSerializerTest.kt: (39, 13): Null can not be a value of a non-null type OrganizationEntryProcessor.Result```

Comment (geekbeast) [reply src/main/kotlin/com/openlattice/organizations/HazelcastOrganizationService.kt:551]: The entry processor in the test wasn't wrapping the result. Should be fixed now.

File: src/main/java/com/openlattice/postgres/ResultSetAdapters.java [hunk 2]
```diff
@@ -40,13 +40,13 @@
 import com.openlattice.collections.EntityTypeCollection;
```
Comment (makosblade) [inline src/main/java/com/openlattice/postgres/ResultSetAdapters.java:40]: I believe this file change is breaking the build

Comment (makosblade) [reply src/main/java/com/openlattice/postgres/ResultSetAdapters.java]: should be good now

**Pull request merged**